### PR TITLE
Improved Quixe processing & testing

### PR DESCRIPTION
**Pull request by jmacdotorg**
Big expansion (and renaming) of the "platforms" test script, and reorganization of test game files. The test now simulates an upload-processing routine on all the test games, and makes sure that the resulting files look as expected.

This PR removes all the already-(fake-)processed files, and masks their previous "content" directories behind a .gitignore line. New files in the entries' "main" directories are ready for processing by the test script.

Quixe entries created with buggy Inform no longer repair themselves via double-base64-encoding. Instead, they modify their HTML to point at the web app's Quixe interpreter, and present a freshly single-encoded JavaScript game file to said interpreter.

### CODE REVIEW

File: IFComp/t/entry_processing.t [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+use FindBin;
+use lib ("$FindBin::Bin/lib");
+use IFCompTest;
+my $schema = IFCompTest->init_schema();
+
+my $entry_directory =
+    Path::Class::Dir->new("$FindBin::Bin/platform_test_entries");
+
+$schema->entry_directory($entry_directory);
+
```
Comment (adherzog) [inline IFComp/t/entry_processing.t]: Is there a compelling reason that this (setting the entry directory on the schema) and the corresponding Entry population below isn't part of IFCompTest?

File: IFComp/t/entry_processing.t [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+}
+
+sub file_contains ($$$) {
+    my ( $entry_id, $filename, $regex ) = @_;
+    my $file = Path::Class::File->new(
+        "$entry_directory/$entry_id/content/$filename" );
+    eval { return $file->slurp =~ $regex; };
+}
+
+diag('Testing naked Z-Code upload...');
```
Comment (adherzog) [inline IFComp/t/entry_processing.t]: Could you change the diag() lines to note()? The only difference is that note() won't display when running under a harness, like prove.

Comment (jmacdotorg) [inline IFComp/t/entry_processing.t]: I was unaware of note()! Thank you, will do.

File: IFComp/t/entry_processing.t (hunk 1 above)
Comment (jmacdotorg) [inline IFComp/t/entry_processing.t]: It probably made sense years ago. Maybe.

I'll see if things blow up if I just move them into the test lib, and if not I'll check that in.

**Comment by adherzog** [comment]
@jmacdotorg A couple tests are failing:

1) The entry id in `controller_Entry.t` needs to be changed from 1 to 111 (now that the Entry table is pre-populated by the test lib.)

2) The test on line 44 of `entry_processing.t` checks for the file `Naked Glulx.gblorb.js` but should check for `Naked glulx.gblorb.js` with a lowercase g.

**Comment by jmacdotorg** [comment]
@adherzog Was literally checking in a fix for (1) as you were typing that, but was blind to (2) because of HFS+. ha

**Pull request merged**